src/Knp/Component/Pager/PaginatorInterface.php

```diff
@@ -21,6 +21,11 @@ interface PaginatorInterface
     public const FILTER_VALUE_PARAMETER_NAME = 'filterValueParameterName';
     public const FILTER_FIELD_WHITELIST = 'filterFieldWhitelist';
     public const DISTINCT = 'distinct';
+    public const PAGE_OUT_OF_RANGE = 'pageOutOfRange';
+
+    public const PAGE_OUT_OF_RANGE_IGNORE = 'ignore';   //do nothing (default)
```

Comment:
please consistently add a space at beginning of comment text

Reply:
I still don't see a space at beginning of comment text (please note that this should be applied to other comments too, so "consistently")